src/main/cpp/mechanisms/ClimberManager/OffState.cpp

```diff
@@ -76,6 +76,9 @@ bool OffState::AtTarget()
 bool OffState::IsTransitionCondition ( bool considerGamepadTransitions )
 {
 	// To get the current state use m_mechanism->GetCurrentState()
-	return false;
+	if (m_mechanism->IsClimbMode() != true) {
```

Comment:
Just return !m_mechanism->IsClimbMode, because you don't have a return false with this current setup